server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -170,7 +175,8 @@
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.context.Scope;
 
-public class TabletServer extends AbstractServer implements TabletHostingServer {
+public class TabletServer extends AbstractServer
+    implements TabletHostingServer, ServerProcessService.Iface {
```

Comment:
A lot of these have redundant interface declarations since it's already declared on the AbstractServer super class. It'd be better to omit ServerProcessService.Iface from the child classes, so it's easier to see where things are coming from in the class hierarchy.

Reply:
That doesn't work with Thrift. You have to put the Thrift interface declarations on the class being used. I agree that they are redundant, but I have run into this many times, especially with overriding behavior of a Thrift server in a test.

Reply:
Hmm, interesting. I wasn't aware there was a problem with that. Then, perhaps it should be put on a separate "handler" class, like the other handlers, and be a member of the AbstractServer?